admin_app/src/app/dashboard/components/TopContentTable.tsx

```diff
@@ -143,7 +145,13 @@ const TopContentTable = ({ rows }: { rows: TopContentData[] }) => {
                   sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
                 >
                   <StyledTableCell component="th" scope="row">
-                    {row.title}
+                    {isDeleted(row.title) ? (
+                      <span style={{ color: "red", fontWeight: "bold" }}>
```

Comment:
Can we use lightGrey instead of red?
And from appColors.lightGrey in `@/utils/index`?

Reply:
yes, i'll change to lightGrey